atlp-rwanda/dynamites-ecomm-fe
# fix bannerSection

### What does this PR do?
- This PR fixes banner section, in oder to avoid more products  be displayed  on banner section

### How to review

- Checkout this branch and run the application.
- Verify that the landing page matches the provided design.

### Screenshots
![Screenshot 2024-07-11 133111](https://github.com/atlp-rwanda/dynamites-ecomm-fe/assets/110623461/47f8fe27-f8c7-4dfc-9cac-a5fa19741a34)
